### ISSUE TITLE
Pre-trained weights

### ISSUE THREAD

**Issue by mnpham0417** [rohitgandikota/unified-concept-editing#1]
Hi, thank you for the great work! May I ask if you are planning to release the pre-trained weights for artistic concepts, objects concepts, and NSFW concept?

**Comment by rohitgandikota** [comment]
Yes, we can release the weights soon. Also, it is super fast to train the models to erase from scratch.

**Comment by Shinyzenith** [comment]
Hi @rohitgandikota Would you mind releasing the pretrained weights? We would love to compare our research against your work.

**Comment by rohitgandikota** [comment]
Hi, we have fixed the repository dependancies and the training efficiency. You should be able to train the models in 1-3 minutes now. Please give it a try!

**Comment by rohitgandikota** [comment]
Released the weights for i2p benchmark unlearning here: https://unified.baulab.info/uce_weights/
